Lib/traceback.py

```diff
@@ -450,34 +450,34 @@ def from_list(klass, a_list):
                 result.append(FrameSummary(filename, lineno, name, line=line))
         return result
 
-    def format_frame(self, frame):
-        """Format the lines for a single frame.
+    def format_frame_summary(self, frame_info):
```

Comment:
Qucik question: any reason to call this `frame_info` instead of `frame_summary` or simply `summar` (I guess this might be a bit ambigous considering `StackSummary` etc. but since we are in `frame_summary()` I am not really sure if that would matter much).

Reply:
I had f_summary and then frame_info. Maybe you're right that frame_summary is better, at least for the signature.